### DISCUSSION TITLE
Benchtop DNA sequencing?

### DISCUSSION THREAD

**Discussion by RobertAKARobin** [Geneticure/geneticure.com#46] [General]
How practical would it be for us to just buy a sequencing system and do it ourselves? e.g. https://www.illumina.com/systems/sequencing-platforms.html Looks like the cheapest one is $20k. 

@Carl362 @snydreric @scottsnyder01

**Comment by scottsnyder01** [comment]
Not at all practical or worth it right now. Might save us $10/test in cost and introduces a mountain of other headaches.

Scott Snyder
952.210.5851
________________________________
From: Robin (Robert) Thomas ***@***.***>
Sent: Friday, September 30, 2022 10:39:04 AM
To: Geneticure/geneticure.com ***@***.***>
Cc: scottsnyder01 ***@***.***>; Mention ***@***.***>
Subject: [Geneticure] Benchtop DNA sequencing? (Discussion #46)


How practical would it be for us to just buy a sequencing system and do it ourselves? e.g. https://www.illumina.com/systems/sequencing-platforms.html Looks like the cheapest one is $20k.

@Carl362<https://github.com/Carl362> @snydreric<https://github.com/snydreric> @scottsnyder01<https://github.com/scottsnyder01>
